src/graphnet/pisa/fitting.py

```diff
@@ -74,6 +75,174 @@ def config_updater(
                 updater.write(configfile)
 
 
+class WeightFitter:
+    def __init__(
+        self,
+        database_path,
```

Comment:
Here and elsewhere we are assuming that databases (as opposed to some other intermediate file format) are being used to interface with PISA. Is that something we are committing to, or just something that works for the time being but might be expanded on down the line?

Reply:
This is something we can expand upon down the line, I think. Currently, GraphNeT is only interfaced with PISA via SQL. To make it work with a new format, we need to update PISA as it relies on a dataloader for each format. See https://github.com/icecube/pisa/tree/master/pisa/stages/data for reference. They are easy to write, so it shouldn't be a big thing.